package.json

```diff
@@ -33,10 +37,11 @@
     "@types/react-dom": "^18",
     "eslint": "^8",
     "eslint-config-next": "14.2.6",
+    "graphql-codegen-plugin-typescript-swr": "0.8.5",
     "pino-pretty": "^11.2.2",
     "postcss": "^8",
     "tailwindcss": "^3.4.1",
     "typescript": "^5"
   },
-  "packageManager": "yarn@4.4.1"
+  "packageManager": "yarn@1.22.22"
```

Comment:
❌  why change yarn version here?

Reply:
what would be the correct way?

Reply:
there's no correct way -- but I'd argue we need a reason to change the pkg manager version. Have you had issues with 4.4.1?

Reply:
I had installed from npm and, for some reason, it came with version 1.22.22. Using 4.4.1. now